scripts/build_anaconda.sh

```diff
@@ -105,8 +105,10 @@ if [[ $GCC_USE_C11 -eq 0 ]]; then
   # gcc compatibility is not tracked by conda-forge, so we track it ourselves
   CAFFE2_PACKAGE_NAME="${CAFFE2_PACKAGE_NAME}-gcc${GCC_VERSION:0:3}"
 fi
-if [[ $BUILD_ENVIRONMENT == *full* ]]; then
-  CAFFE2_PACKAGE_NAME="${CAFFE2_PACKAGE_NAME}-full"
+if [[ "$(uname)" != 'Darwin' ]]; then
```

Comment:
Is this intentional?

Reply:
This is a small fix to a separate bug

Reply:
What bug? XD

Reply:
The mac conda builds have been named caffe2-gcc for the past few days